pokemon/make_db.py

```diff
@@ -0,0 +1,101 @@
+#from pokemon.convert import handle_image_conversion
```

Comment:
Can you walk me through why this is presented as a new file?

Reply:
As below; I'll move make_db.py back to scripts and alter references back to the way they were.